src/models/Journey.tsx

```diff
@@ -0,0 +1,13 @@
+
+export type Journey = {
+    arrivalTime: string;
+    departureTime: string;
+    destinationStation: JourneyStation;
```

Comment:
Why does this not have an originStation?

Reply:
added an originStation